libs/map-editor-types/package.json

```diff
@@ -2,15 +2,15 @@
   "name": "@workadventure/map-editor-types",
   "version": "1.0.0",
   "description": "Map Editor commands types",
-  "main": "src/index.ts",
+  "main": "./src/index.ts",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "WorkAdventure",
   "license": "ISC",
   "dependencies": {
     "@workadventure/tiled-map-type-guard": "^2.0.4",
-    "@workadventure/math-utils": "link:../math-utils"
+    "typescript": "^4.3.5"
```

Comment:
Do we need this as a strong dependency? Can't we have it as a dev-dependency instead (otherwise this will bundle typescript in the final images where it is not needed)